### PR TITLE
Prepare Xtext/Xtend 2.35 release notes

### PR DESCRIPTION
**Pull request by LorenzoBettini**
See https://github.com/eclipse/xtext/issues/2948

### CODE REVIEW

File: xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Call to Action: Secure the future maintenance of Xtext & Xtend
+
+As you might have recognized, the number of people contributing to Xtext & Xtend on a regular basis has declined over the past years and so has the number of contributions. At the same time the amount of work for basic maintenance has stayed the same or even increased with the new release cadence of Java and the Eclipse simultaneous release. Briefly: The future maintenance of Xtext & especially Xtend is at risk, at least in the current form and as part of the Eclipse Simrel. If you care, please join the discussion in [https://github.com/eclipse/xtext/issues/1721](https://github.com/eclipse/xtext/issues/1721).
+
+
+## Upgrades
+
+Cedits
+
+see Xtext release notes
```
Comment (tivervac) [inline xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md:20]:
```suggestion
## Credits

See Xtext release notes
```

File: xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  Xtend 2.35.0 Release Notes
+date:   2024-04-22
```
Comment (cdietrich) [inline xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md:4]: ususally we pick the tuesday of the rc1 week as file name and release date

File: xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+categories: releasenotes
+published: true
```
Comment (cdietrich) [inline xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md:6]: should be falsse

File: xtext-website/_posts/releasenotes/2024-04-22-version-2-35.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  Xtext 2.35.0 Release Notes
+date:   2024-02-27
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2024-04-22-version-2-35.md:4]: adapt date (also in filename)

File: xtext-website/_posts/releasenotes/2024-04-22-version-2-35.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+categories: releasenotes
+published: true
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2024-04-22-version-2-35.md:6]: false

File: xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md (hunk 2 above)
Comment (LorenzoBettini) [reply xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md:4]: @cdietrich where do I find such information?

Comment (cdietrich) [reply xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md:4]: https://github.com/eclipse-simrel/.github/tree/main/wiki/SimRel

**Comment by LorenzoBettini** [comment]
@cdietrich updated!

**Review by cdietrich — APPROVED** [review]

**Comment by cdietrich** [comment]
exactly (main branch is dev, published branch is the live one)

**Comment by miklossy** [comment]
@cdietrich @LorenzoBettini currently the file ends with: `2-35.md`. To be conform to the other files, it should end with `2-35.0.md`, shouldn't it?

**Comment by cdietrich** [comment]
it shoudl be as its for the other files

**Comment by LorenzoBettini** [comment]
@miklossy sorry my fault :(
could you please fix it?

**Comment by miklossy** [comment]
yes, after https://github.com/eclipse/xtext-website/pull/34 has been merged, I will create a follow-up PR with the file renaming :-)

**Comment by miklossy** [comment]
I created the PR https://github.com/eclipse/xtext-website/pull/35

**Pull request merged**